CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
 ### unreleased
 
+### 0.0.4 (2019-10-20)
+
+* Update to use progress reporting as defined in LSP 3.15. The VSCode
+  extension has also been updated and should now be making use of
+  this.
+* Properly declare that we should support code actions. This helps
+  with some clients that rely on this information to enable code
+  actions (thanks @jacg).
+* Fix a race condition caused by sharing the finder cache between
+  concurrent compilations.
+* Avoid normalizing include dirs. This avoids issues where the same
+  file ends up twice in the module graph, e.g., with different casing
+  for drive letters.
+* More code actions.
```

Comment:
Could we be more precise?

Reply:
I’ll try to expand it tomorrow.

Reply:
Actually it turns out that all of those changes were before the last release so I’ve removed this. Should have looked closer at the commits initially.